### PR TITLE
[Dependency Bump] Resolve log4j vulnerability

### PR DESCRIPTION
**Pull request by katukota-google**
Bumped hadoop versions and excluded a few dependencies.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -192,6 +203,10 @@
           <groupId>org.apache.zookeeper</groupId>
           <artifactId>zookeeper</artifactId>
         </exclusion>
+        <exclusion>
```
Comment (dli357) [inline pom.xml:199]: What is the purpose of this dependency exclusion?

Comment (katukota-google) [reply pom.xml:199]: [INFO] io.cdap.cdap.test:perf-test:jar:1.0.0
[INFO] \- org.apache.kafka:kafka_2.11:jar:0.8.2.2:compile
[INFO]    \- com.101tec:zkclient:jar:0.3:compile
[INFO]       \- log4j:log4j:jar:1.2.14:compile

Comment (dli357) [reply pom.xml:199]: Maybe we can just exclude only log4j?

**Review by dli357 — APPROVED** [review]
LGTM, pending one comment

File: integration-test-remote/pom.xml [hunk 2]
```diff
@@ -84,10 +84,18 @@
...
         </exclusion>
+        <exclusion>
+          <groupId>log4j</groupId>
+          <artifactId>log4j</artifactId>
+        </exclusion>
         <exclusion>
           <groupId>asm</groupId>
           <artifactId>asm</artifactId>
         </exclusion>
+        <exclusion>
```
Comment (dli357) [inline integration-test-remote/pom.xml:95]: nit: Double exclusion

**Review by dli357 — APPROVED** [review]
One nit comment, otherwise LGTM. Thanks!

**Pull request closed without merging**